vizro-core/src/vizro/static/css/table.css

```diff
@@ -52,6 +52,37 @@
   padding: 4px 0;
 }
 
+#right-side .dash-table-container .previous-next-container button,
```

Comment:
We may want to also apply some of this to the pagination of the AgGrid

Reply:
Would be good if you could double-check that in your CSS PR. Just add pagination and see if the CSS is messed up there. I just squeezed this in here, as it was obviously wrong in the sample screen